test/nvidia_gpu/metrics_nvidia_gpu_window_test.go

```diff
@@ -24,7 +24,14 @@ const (
 	numberofWindowsAppendDimensions = 1
 )
 
-var expectedNvidiaGPUWindowsMetrics = []string{"Memory % Committed Bytes In Use", "nvidia_smi utilization_gpu", "nvidia_smi utilization_memory", "nvidia_smi power_draw", "nvidia_smi temperature_gpu"}
+var (
+	envMetaDataStrings              = &(environment.MetaDataStrings{})
```

Comment:
This conflicts with the declaration in the linux test doesn't it?

Reply:
Yes you are right. Let's me confirm that real quick.